src/java/fr/paris/lutece/portal/business/user/menu/AccessibilityModeAdminUserMenuItemProvider.java

```diff
@@ -64,9 +66,10 @@ protected boolean isItemProviderInvoked( HttpServletRequest request )
     public AdminUserMenuItem getItem( HttpServletRequest request )
     {
         AdminUser user = AdminUserService.getAdminUser( request );
+        ISecurityTokenService securityTokenService = CDI.current( ).select( ISecurityTokenService.class ).get( );
```

Comment:
Do not use static calls; inject the service as a property since this class is a CDI bean.
@Inject 
private ISecurityTokenService _securityTokenService;